execution_environments/how_to_build_and_test_first_ee.rst

```diff
@@ -0,0 +1,182 @@
+List the files you created at previous steps:
+
+.. code-block:: yaml
+
+  ---
+  version: 3
+
+  dependencies:
+    galaxy: requirements.yml
+    python: requirements.txt
+
+5. Create the ``demo`` build context directory:
+
+.. code-block:: bash
+
+  $ mkdir demo
+
+Besides container artifacts, it will contain a ``Containerfile`` generated by ``ansible-builder``,
+our inventories and playbooks we will add later to test the EE with ``ansible-runner``.
+
+6. Build the EE container image called ``postgresql_ee`` using the ``demo`` directory as a build context (if you use docker, add the ``--container-runtime docker`` argument to the command):
+
+.. code-block:: bash
+
+  $ ansible-builder build --tag postgresql_ee --context demo
+
+7. You now can see the container image in the list of images. Assuming you use podman:
+
+.. code-block:: bash
+
+  $ podman image list
+
+  localhost/postgresql_ee          latest      2e866777269b  6 minutes ago  1.11 GB
+
+8. Inspect the ``Containerfile`` file (``Dockerfile`` if you use docker) in the ``demo`` directory to see the EE container image configuration.
+
+The created image represents an :ref:`Ansible control node<terminology>` which now contains:
+
+* Python
+* ansible-core
+* ansible-runner
+* the community.postgresql collection
+* the psycopg2-binary Python package
+
+See the :ref:`Running your EE in command line<run_firt_ee>` section to learn how to test the EE you have just created.
+
+**TBD: Add links referring to overview pages for the files above when created.**
+
+**TBD: Add a link to the builder's overview page when created.**
+
+For more information about Ansible Builder, see the :ref:`Ansible Builder Overview<ADD_LINK_WHEN_WRITTEN>` document.
+
+
+.. _run_first_ee:
+
+Running your EE in command line
+===============================
+
+Here, we will test the EE you created in the :ref:`Building your first EE<build_first_ee>` section against the localhost and a remote target.
+
+You are still in the ``postgresql_ee`` directory.
+
+Create two other directories for our inventories and playbooks:
+
+.. code-block:: bash
+
+  $ mkdir demo/{inventory,project}
+
+If you use ``docker``:
+
+* add the ``--process-isolation-executable docker`` argument to the ``ansible-runner`` commands below
+* the EE image name used in the commands can differ
+
+
+Run against localhost
+---------------------
+
+1. Create the ``test_localhost.yml`` playbook file in the ``demo/project`` directory with the following content:
+
+.. code-block:: yaml
+
+  ---
+  - hosts: localhost
+    become: yes
+    gather_facts: yes
+    tasks:
+    - name: Print facts
+      ansible.builtin.debug:
+        msg: '{{ ansible_facts }}'
+
+2. Run the playbook inside the EE container with ``ansible-runner``:
+
+.. code-block:: bash
+
+  $ ansible-runner run demo --playbook test_localhost.yml --container-image localhost/postgresql_ee --process-isolation
```

Comment:
Why are you using ansible-runner and not ansible-navigator? I was assuming that ansible-runner is more like a low-level tool that is used by AWX and ansible-navigator, and that end-users should use ansible-navigator. (I'm not sure where I got this impression from though...)

Reply:
> Why are you using ansible-runner and not ansible-navigator? I was assuming that ansible-runner is more like a low-level tool that is used by AWX and ansible-navigator, and that end-users should use ansible-navigator. (I'm not sure where I got this impression from though...)

Probably because i haven't read the navigator's docs yet;) It's in my list, i'll prioritize it, thanks